_includes/i18n-as-script.html

```diff
@@ -0,0 +1,11 @@
+{% comment %} get the current translations from Jekyll, to be used in scripts {% endcomment %}
+<script>
+  window.i18n = {
+    url_base: '{{url_base}}',
+    active_language: '{{vars_active_language.tag}}',
+    search_placeholder: '{{ i18n.search_placeholder }}',
+    search_by_algolia: '{{ i18n.search_by_algolia }}',
+    search_by_read_more: '{{ i18n.search_by_read_more }}',
```

Comment:
@Haroenv - It also prevents someone from performing XSS via translations (eg. if someone were to somehow enter the translation as `foo'; window.location = 'http://badsite.com/';`) 😃